app/templates/components/events/view/export/download-zip.hbs

```diff
@@ -10,23 +10,28 @@
 <div class="ui container"></div>
 <div class="ui form">
   <div class="field">
-    {{ui-checkbox class='toggle' label=(t 'Image')}}
+    {{ui-checkbox class='toggle' label=(t 'Image') checked=dataImage onChange=(action (mut dataImage))}}
```

Comment:
How exactly are you passing in the variable `dataImage` and the rest to the component from the controller ?